.gitignore

```diff
@@ -1,3 +1,24 @@
-**/node_modules
-
 .DS_Store
+node_modules
+/dist
+package-lock.json
+
+
+# local env files
+.env.local
+.env.*.local
+
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
```

Comment:
I am not sure why we need to ignore those three logs since we are only use npm to manage the libraries. Yarn and npm basically would have the same functions. Are we gonna use something that could only be imported by Yarn?

Reply:
These were automatically put in the .gitignore file when I created the vue app. As far as I can tell, these files do not exist and are there as a precaution in case we do end up creating these files. Since that will almost definitely not happen, you can delete them if you would like.